website/config/routes.js

```diff
@@ -68,6 +68,14 @@ module.exports.routes = {
     }
   },
 
+  'r|/((device-management|security|releases|engineering|guides|announcements|use-cases)/(.+))$|': {
+    skipAssets: false,
+    action: 'articles/view-basic-article',
+    locals: {
+      currentPage: 'articles',
+    }
+  },// handles /device-management/foo, /security/foo, /releases/foo, /engineering/foo, /guides/foo, /announcements/foo, /use-cases/foo
```

Comment:
What happens if someone navigates to `/device-management`?

Reply:
They would see a 404 page, it would not be handled by this route.

Reply:
:+1: Ok great, that works for now!